### PR TITLE
Merge master into release

### PR DESCRIPTION
**Pull request by DustinCampbell**

### CODE REVIEW

File: coreclr-debug/NuGet.config [hunk 1]
```diff
@@ -4,5 +4,7 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <!-- This dependency is not present in the release branch -->
+    <add key="coreclrdebug" value="https://www.myget.org/F/coreclr-debug/api/v3/index.json" />
```
Comment (gregg-miskelly) [inline coreclr-debug/NuGet.config]: Did this line actually flow back? If so, it shouldn't have.

**Comment by gregg-miskelly** [comment]
Otherwise LGTM

File: coreclr-debug/NuGet.config (hunk 1 above)
Comment (DustinCampbell) [inline coreclr-debug/NuGet.config]: It certainly did.

Comment (DustinCampbell) [inline coreclr-debug/NuGet.config]: How do we fix that? I don't want to cherry-pick the world.

Comment (gregg-miskelly) [inline coreclr-debug/NuGet.config]: Okay, we will need to back that out then before we make a .vsix.

Comment (gregg-miskelly) [inline coreclr-debug/NuGet.config]: I think you can commit this PR as-is and I will fix it once you are done. Alternatively I can fix it in master. Either way is fine with me.

Comment (DustinCampbell) [inline coreclr-debug/NuGet.config]: Merged. Feel free to fix.

**Pull request merged**